static/app/utils/queryString.tsx

```diff
@@ -166,6 +167,16 @@ export function decodeBoolean(
   return fallback;
 }
 
+export const parseAsSort = createParser({
```

Comment:
I added the parser here in the query string utils which seemed like a good home because it already contains the decoders. But LMK if you think it should be in a separate file